### PR TITLE
G-API: Updates G-API tutorial using normalize kernel

### PR DESCRIPTION
**Pull request by smirnov-alexey**
This pullrequest changes text and pictures used in tutorial along with sample code

### CODE REVIEW

**Comment by smirnov-alexey** [comment]
Attached a tutorial page with changes
[tutorial_gapi_anisotropic_segmentation.html.zip](https://github.com/opencv/opencv/files/2841247/tutorial_gapi_anisotropic_segmentation.html.zip)

File: doc/tutorials/gapi/anisotropic_segmentation/porting_anisotropic_segmentation.markdown [hunk 1]
```diff
@@ -163,7 +161,7 @@ or viewed instantly with `xdot` command (please refer to your
 distribution/operating system documentation on how to install these
 packages).
 
-![Anisotropic image segmentation graph](pics/segm.gif)
+![Anisotropic image segmentation graph](pics/segm.png)
```
Comment (alalek) [inline doc/tutorials/gapi/anisotropic_segmentation/porting_anisotropic_segmentation.markdown]: > segm.gif

Is file still used? If not, remove it too.

File: doc/tutorials/gapi/anisotropic_segmentation/porting_anisotropic_segmentation.markdown [hunk 2]
```diff
@@ -367,16 +365,16 @@ Fluid backend. Now it looks like this:
...
+code, without modifying the graph itself! It is a ~2.4X improvement of
+the previous G-API result, and ~1.6X improvement of the original OpenCV
 version.
 
 Let's also examine how the internal representation of the graph now
 looks like. Dumping the graph into `.dot` would result into a
 visualization like this:
 
-![Anisotropic image segmentation graph with OpenCV & Fluid kernels](pics/segm_fluid.gif)
+![Anisotropic image segmentation graph with OpenCV & Fluid kernels](pics/segm_fluid.png)
```
Comment (alalek) [inline doc/tutorials/gapi/anisotropic_segmentation/porting_anisotropic_segmentation.markdown]: The same.

File: doc/tutorials/gapi/anisotropic_segmentation/porting_anisotropic_segmentation.markdown (hunk 1 above)
Comment (smirnov-alexey) [reply doc/tutorials/gapi/anisotropic_segmentation/porting_anisotropic_segmentation.markdown]: removed

File: doc/tutorials/gapi/anisotropic_segmentation/porting_anisotropic_segmentation.markdown (hunk 2 above)
Comment (smirnov-alexey) [reply doc/tutorials/gapi/anisotropic_segmentation/porting_anisotropic_segmentation.markdown]: removed

**Comment by smirnov-alexey** [comment]
@dmatveev,  it's +1.5mb

**Comment by smirnov-alexey** [comment]
Now it's +90kb from gifs (graphs) and +300kb from png (massif)

**Comment by alalek** [comment]
Squash commits to remove intermediate files from Git history.

**Comment by dmatveev** [comment]
👍 well done!

**Pull request merged**